yt_dlp/extractor/southpark.py

```diff
@@ -109,6 +109,48 @@ def _get_feed_query(self, uri):
         return
 
 
+class SouthParkLatIE(SouthParkIE):
+    IE_NAME = 'southpark.lat'
+    _VALID_URL = r'https?://(?:www\.)?(?P<url>southpark\.lat/(?:(en/(videoclip|collections|episodes|video-clips))|(episodios|videoclip|collections))/(?P<id>(?P<unique_id>.+?)/.+?)(?:\?|#|$))'
```

Comment:
is this not sufficient?

```suggestion
    _VALID_URL = r'https?://(?:www\.)?(?P<url>southpark\.lat/(?:en/)?(?:video-?clips?|collections|episod(?:e|io)s)/(?P<id>[^/#?]+)'
```

Reply:
@pukkandan check this: https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/southpark.py   the code from southpark.de was already like this, so I just tried to replicate it.
![image](https://user-images.githubusercontent.com/17991404/172449026-d0a60c4c-76fe-4deb-88a4-d01ba369ac26.png)

Reply:
Let me get this straight - You don't want to make this change because there is an 8yo regex in the file that is similar to your current code? Old and new code are held to different standards. I will not merge this unnecessarily complicated regex unless you have some justification of why it needs to be so complicated

Reply:
@pukkandan Done, I already fixed the regex with your suggestion. check now please.